netplugin/netd.go

```diff
@@ -456,6 +456,18 @@ func createContainerEpOper(netPlugin *plugin.NetPlugin, contUUID, contName strin
 				log.Errorf("Endpoint creation failed. Error: %s", err)
 				return err
 			}
+
+			err = operEp.Read(epID)
```

Comment:
do we need to read operEp again? it looks like we already read operEp at line 439